src/NServiceBus.Core.Tests/Unicast/StartAndStoppablesRunnerTests.cs

```diff
@@ -141,6 +142,17 @@ public void Should_throw_if_startable_fails_asynchronously()
             Assert.True(startable2.Stopped);
         }
 
+        [Test]
+        public void Should_throw_friendly_invalidoperationexception_when_start_returns_null()
```

Comment:
where is the `stop_returns_null` test?

I'm not a fan of the bool flags. I'd rather prefer to implementations.